.github/workflows/unit-tests-diff-code.yaml

```diff
@@ -0,0 +1,72 @@
+name: Unit Tests (Go diff)
+run-name: Unit Tests (Go diff) - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+env:
+  NUM_TRIES: 30
+  DIFFTEST: build.assets/tooling/bin/difftest
+  EXCLUDE: operator/**/*,tool/tsh/**/*integration/**/*,build.assets/**/*
+
+jobs:
+  test:
+    name: Unit Tests (Go, diff)
+    runs-on: ubuntu-22.04-32core
+
+    permissions:
+      contents: read
+      packages: read
+
+    container:
+      image: ghcr.io/gravitational/teleport-buildbox:teleport12
+      env:
+        TELEPORT_ETCD_TEST: yes
```

Comment:
If we're excluding integration/ do we need to run etcd? Do any of the unit tests require etcd?

Reply:
lib/backend/etcdbk/etcd_test.go